integration-tests/tests/run-analysis-azure.js

```diff
@@ -0,0 +1,89 @@
+// This test is owned by the Interactive Analysis (IA) Team.
```

Comment:
Wondering if we can make this new file `.ts` without any issues? If its trivial I'd like to see new files being typescript

Reply:
i'll try!